src/overlays/actors/ovl_En_Tana/z_en_tana.c

```diff
@@ -33,6 +33,11 @@ const ActorInit En_Tana_InitVars = {
 static const char* sShelfTypes[] = {
     "木の棚", // "Wooden Shelves"
     "石の棚", // "Stone Shelves"
+#ifndef AVOID_UB
+// missing string for second stone type
+#else
```

Comment:
before someone comments on this, i tried indenting it but the almighty formatter said no